subcommands/keys/rotate_root.go

```diff
@@ -48,6 +48,13 @@ func doRotateRoot(cmd *cobra.Command, args []string) {
 	subcommands.DieNotNil(err)
 
 	if initialRotation {
+		if _, err := os.Stat(credsFile); err == nil {
+			fmt.Printf("ERROR: %v already exists\n", credsFile)
+			fmt.Printf("WARNING: do not just remove %v, could contain other keys\n", credsFile)
+			fmt.Println("Make sure to run fioctl keys rotate-root --initial /absolute/path/to/non.existent.file.tgz")
+			os.Exit(1)
```

Comment:
i don't want to turn this into bikeshedding but we could probably keep it more concise and consistent with:
```
subcommands.DieNotNil(errors.New("Destination file exists. Please make sure you aren't accidentally overwriting another factory's keys"))
```

Reply:
Done.